scripts/populate_prices.py

```
@@ -676,75 +756,144 @@ async def populate_itad_prices(
+                                "data": {
+                                    "lowest_price": "0",
+                                    "lowest_price_formatted": "Free",
+                                    "shop_name": "Steam",
+                                },
+                                "method": "steam_fallback",
+                                "game_name": game_data.get("name"),
+                            }
+            else:
+                completed = 0
+                for coro in asyncio.as_completed(tasks):
+                    app_id, success, game_data, source = await coro
```

Ruff (0.15.7)

[warning] Unpacked variable `source` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)